runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java

```diff
@@ -971,10 +971,8 @@ public void setXWalkWebChromeClient(XWalkWebChromeClient client) {
         mContent.setXWalkWebChromeClient(client);
     }
 
-    /**
-     * @hide
-     */
-    public void setDownloadListener(DownloadListener listener) {
+    @XWalkAPI
+    public void setDownloadListener(XWalkDownloadListenerInternal listener) {
```

Comment:
Please help to add document:
/**
* Registers the interface to be used when content can not be handled by
* the rendering engine, and should be downloaded instead. This will replace
* the current handler.
* @param listener an implementation of XWalkDownloadListenerInternal 
* @since 5.0
*/